webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/db/Doobie.scala

```diff
@@ -102,40 +102,47 @@ class Doobie(datasource: DataSource) {
 
 object Doobie {
 
-  implicit val slf4jDoobieLogger: LogHandler = {
-    object DoobieLogger extends Logger {
-      override protected def _logger = LoggerFactory.getLogger("sql")
+  implicit val slf4jDoobieLogger: LogHandler[IOResult] = {
+    object DoobieLogger extends NamedZioLogger {
+      val loggerName = "sql"
     }
 
-    LogHandler {
-      // we could log only if exec duration is more than X ms, etc.
-      case doobie.util.log.Success(s, a, e1, e2) =>
-        val total = (e1 + e2).toMillis
-        val msg   = s"""Successful Statement Execution [${total} ms total (${e1.toMillis} ms exec + ${e2.toMillis} ms processing)]
-                     |  ${s.linesIterator.dropWhile(_.trim.isEmpty).mkString("\n  ")}
-                     | arguments = [${a.mkString(", ")}]
+    new LogHandler[IOResult] {
```

Comment:
doesn't it change a call site somewhere (adding a `.runNow` ?) ? Or is it that `LogHandler` changed from `LogEvent => Unit` to something else its signature ?

Reply:
It changes to LogEvent => M[Unit] Where M is your effect

Reply:
(it's ok, boobie changed signature)